plat/common/memory.c

```diff
@@ -133,7 +120,7 @@ void *ukplat_memregion_alloc(__sz size, int type, __u16 flags)
 		alloc_mrd.len = desired_sz;
 		alloc_mrd.pg_count = PAGE_COUNT(desired_sz);
 		alloc_mrd.type = type;
-		alloc_mrd.flags = flags | UKPLAT_MEMRF_MAP;
+		alloc_mrd.flags = flags;
 
 		bi = ukplat_bootinfo_get();
```

Comment:
I know I wrote this :laughing: but ditto, this is a crash reason tbh.

Reply:
In principle I agree, but tgaking into account earlier comments on the same topic, and the number places in the code base where we actually check the return value of `ukplat_bootinfo_get()`, I would rather if we came up with a common approach that is also expressed on the interface of this function, rather than ad-hoc requests on reviews. Either way, I would leave this as is given that it was not changed in this PR and the error is propagated to the caller anyway.

Reply:
> The **unbelievably handsome and charismatic** author may have just changed his mind 🤔, I am joking (about the latter). In all seriousness though, I believe these changes were back when I barely had any familiarity with this code I guess. I am fine with leaving as is and coming back when we rearch ukboot.

Ditto. I am fine with leaving for later.